lapack.go

```diff
@@ -108,9 +108,9 @@ type SVDJob byte
 
 const (
 	SVDAll       SVDJob = 'A' // Compute all singular vectors
-	SVDInPlace          = 'S' // Compute the first singular vectors and store them in provided storage.
-	SVDOverwrite        = 'O' // Compute the singular vectors and store them in input matrix
-	SVDNone             = 'N' // Do not compute singular vectors
+	SVDInPlace   SVDJob = 'S' // Compute the first singular vectors and store them in provided storage.
```

Comment:
Oops, sorry. Apparently I don't quite know how constant declarations work in Go.